cfme/tests/containers/test_relationships_tables.py

```diff
@@ -202,25 +219,24 @@ def test_images_rel(provider, rel, detailfield):
         table_locator="//div[@id='list_grid']//table")
     ui_images = [r.name.text for r in list_tbl_image.rows()]
 
-    for name in ui_images:
-        img = Image(name, provider)
-        navigate_to(img, 'Details')
+    img = Image(random.choice(ui_images))
```

Comment:
Choice instead of shuffle because of the potential for a large number of images?